exchanges/hitbtc/hitbtc_websocket.go

```diff
@@ -45,6 +48,13 @@ func (h *HitBTC) WsConnect() error {
 	}
 
 	go h.WsHandleData()
+	if h.AuthenticatedAPISupport {
+		err = h.wsLogin()
+		if err != nil {
+			return err
```

Comment:
There exists a websocket consistency issue accross the codebase where if a wsAuth attempt fails, we'll return an err and negate any futher features like generating the default ws subscriptions for all exchanges. The following changes will need to be made:

1. A new variable to differentiate whether we have websocket API support enabled so disabling it here won't impact the REST side in case of a transient issue specifically impacting WS.
2. Log the err so the user is informed rather then returning it.
3. Continue on with public streaming endpoints (GeneratingDefaultSubscriptions).